src/validation/custom.validation.ts

```diff
@@ -89,3 +90,23 @@ export const checkAtLeastOneFieldHasValue = (errMsg: string, ...fields: any[]) =
   }
   throw new Error(errMsg);
 };
+
+export const checkMandatoryTrustFields = (nameErrMsg, dateErrMsg, trustsJson: string) => {
+  const trusts: trustType.Trust[] = JSON.parse(trustsJson);
```

Comment:
just thinking, if you enter un-parseable json, I imagine JSON.parse will throw an error here, and error thrown from this function will get displayed on the screen. So I think you'd have to wrap the JSON.parse line in it's own try-catch and log the parse error using logger.error and throw a 'friendly' error message - 'invalid json' or something.

Reply:
ROE-789 will deal with un-parseable json

Reply:
ok, just be aware that if that work had already been tested, will need to be retested when we add this check in